__tests__/create-token.test.js

```diff
@@ -0,0 +1,71 @@
+import TestUtils from "./test-utils";
+
+describe("create-token api", () => {
+  it("should return 200 with a valid body", async () => {
+    const response = await TestUtils.request
+      .post("/wallet/create-token")
+      .send({
+        name: "stub_token",
+        symbol: "03",
+        amount: 1,
+      })
+      .set({ "x-wallet-id": TestUtils.walletId });
+    expect(response.status).toBe(200);
+    expect(response.body.hash).toBeDefined();
+  });
+
+  it("should create a token with amount as string", async () => {
+    const response = await TestUtils.request
+      .post("/wallet/create-token")
+      .send({
+        name: "stub_token",
+        symbol: "03",
+        amount: "1",
+      })
+      .set({ "x-wallet-id": TestUtils.walletId });
+    expect(response.status).toBe(200);
+    expect(response.body.hash).toBeDefined();
+  });
+
+  it("should not create a token without the required parameters", async () => {
+    ["name", "symbol", "amount"].forEach(async (field) => {
+      const token = {
+        name: "stub_token",
+        symbol: "03",
+        amount: 1,
+      };
+      delete token[field];
+      const response = await TestUtils.request
+        .post("/wallet/create-token")
+        .send(token)
+        .set({ "x-wallet-id": TestUtils.walletId });
+      expect(response.status).toBe(400);
+      expect(response.body.success).toBeFalsy();
+    });
+  });
+
+  it("should receive an error when trying to do concurrent create-token (lock/unlock behavior)", async () => {
```

Comment:
Can we create a test to try to run `/wallet/create-token` and `/wallet/simple-send-tx` in the same time? It should block the second request as well.

Reply:
Done